src/triggers/default_triggers.py

```diff
@@ -37,8 +37,9 @@
         ),
         "let_service_choose_network_type": Trigger(
```

Comment:
I think the name should be changed now. Maybe "default_to_sdn_for_ocp_under_4.12"

but another question, why do we need this trigger? if network-type not provided the service will probably choose sdn by itself

Reply:
Not on versions below 4.12.
I think that @danmanor added it a while back.

Reply:
In that case the network type will be nil and will not be set

Reply:
p.s I updated the trigger name

Reply:
but it gotta choose some network type eventually, no?
I mean, it will not get blocked on validations or something like it AFAIK

Reply:
I think that openshift installer is choosing its default value

Reply:
IIRC we wanted to check if clusters with `openshift_version` >= 4.12 are set correctly with network type = `OVNKubernetes`, but test-infra was setting it to be `OpenshiftSDN` anyway, so we added this trigger to let the backend decide the network type when the version is >= 4.12. I am not sure I understand why the network type is set to None for versions < 4.12